core/templates/dev/head/pages/exploration_player/audio_bar_directive.html

```diff
@@ -1,4 +1,4 @@
-<div class="audio-header">
+<div class="audio-header" ng-if="languagesInExploration.length > 0">
```

Comment:
Just to double-check: does this take into account both manually-translated and autogenerated languages?

Reply:
Yes. This value is set in AudioTranslationManagerManagerService.js using a priority sequence mentioned there. I think it takes into account all situations except when an auto-generated language is not available which is what we were running into. I tested it across 3 but not with a manually-translated language. Is there an exploration with a translation which I can test this on?

Reply:
Yes I tested it with the Audio Test exploration on the admin page. It shows the bar if a manual translation is available and it works fine.